jest.setup.ts

```diff
@@ -3,21 +3,26 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import "@testing-library/jest-dom";
+import "@testing-library/jest-dom/jest-globals";
+import { TestingLibraryMatchers } from "@testing-library/jest-dom/matchers";
 import { TextEncoder } from "util";
 import { setProjectAnnotations } from "@storybook/react";
-import { toHaveNoViolations } from "jest-axe";
-import { expect } from "@jest/globals";
 import { defaultFallbackInView } from "react-intersection-observer";
 import failOnConsole from "jest-fail-on-console";
-
 import * as globalStorybookConfig from "./.storybook/preview";
+import "jest-axe/extend-expect";
+
+declare module "expect" {
```

Comment:
Let me know if I've understood this correctly. I also submitted a PR upstream so we hopefully won't need this later.

```suggestion
// `@types/jest-axe` doesn't extend `expect` when imported from `@jest/globals`.
// This manually sets the correct type. Note that it has to extend the
// TestingLibraryMatchers to avoid overriding those.
// Also see
// https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67495
declare module "expect" {
```

Reply:
Nice! Thanks for opening that PR.